src/ttsim/plot/dag/tt.py

```diff
@@ -202,23 +223,13 @@ def _get_tt_dag_with_node_metadata(
     return selected_dag
 
 
-def convert_all_nodes_to_callables(
-    env: SpecEnvWithoutTreeLogicAndWithDerivedFunctions,
-) -> SpecEnvWithoutTreeLogicAndWithDerivedFunctions:
-    return {
-        qn: dummy_callable(obj=n, leaf_name=dt.tree_path_from_qname(qn)[-1])
-        if not callable(n)
-        else n
-        for qn, n in env.items()
-    }
-
-
 def _get_node_descriptions(
-    env: SpecEnvWithoutTreeLogicAndWithDerivedFunctions,
+    specialized_environment: PolicyEnvironment,
```

Comment:
This is very confusing naming (=namespace, but PolicyEnvironment annotation...). I have no idea what this should be.